packages/hardhat-zksync-verify/src/zksync-block-explorer/verification-status-response.ts

```diff
@@ -0,0 +1,52 @@
+enum VerificationStatusEnum {
+    successful = 'successful',
+    failed = 'failed',
+    queued = 'queued',
+    in_progress = 'in_progress',
```

Comment:
Should we maybe keep this in camelCase?

```suggestion
    inProgress = 'in_progress',
```